opencsp/app/sofast/test/test_ImageAcquisitionAbstract.py

```diff
@@ -0,0 +1,99 @@
+import os
```

Comment:
Should this entire file (test_ImageAcquisitionAbstract.py) be moved to common/lib/camera/test to live next to ImageAcquisitionAbstract.py?
(To be clear, I still think ImageAcquisition_no_camera.py should live here because it's used to test Sofast stuff fyi).

Reply:
Yes. Moved!